eos-knowledge-lib.modules

```diff
@@ -199,6 +199,7 @@ use the latest stable release branch from GNOME if you prefer that.
       <dep package="gjs"/>
       <dep package="glib"/>
       <dep package="gtk+"/>
+      <dep package="emeus"/>
```

Comment:
The only thing missing was a dependency on meson system package (required for building emeus), after I added it manually, I was able to successfully follow all steps of the contributing file.